code/datums/gamemode/dynamic/dynamic.dm

```diff
@@ -658,7 +659,9 @@ var/stacking_limit = 90
 					continue
 			dead_players.Add(M)//Players who actually died (and admins who ghosted, would be nice to avoid counting them somehow)
 	
-	if(!living_players.len && world.time > 15 MINUTES) //if nobody is around and alive in the current round and enough time has passed
+	if(living_players.len) //if anybody is around and alive in the current round
+		last_time_of_population = world.time
+	if(last_time_of_population && world.time - last_time_of_population > 5 MINUTES && world.time > 15 MINUTES) //if enough time has passed without it
```

Comment:
by making this is an `else if`, you can save one subtraction calculation given that this should never be checked if we just confirmed there are players

it's really minor but no reason not to